src/java/org/apache/cassandra/concurrent/AbstractLocalAwareExecutorService.java

```diff
@@ -150,7 +149,7 @@ class FutureTask<T> extends SimpleCondition implements Future<T>, Runnable
 
         public FutureTask(Callable<T> callable)
         {
-            this.callable = callable;
+            this.callable = Tracers.wrap("FutureTask#run", callable);
```

Comment:
For my own future ref: https://github.com/palantir/tracing-java/blob/4.23.0/tracing/src/main/java/com/palantir/tracing/Tracers.java#L179